### PR TITLE
OCaml 4.11 support

### PR DESCRIPTION
**Pull request by kit-ty-kate**
OCaml 4.11 is around the corner and camlp4 is not compatible with it. This PR should fix it (I heard @ygrek took the maintenance over and it was very quick fix)

### CODE REVIEW

File: configure [hunk 1]
```diff
@@ -45,7 +45,7 @@ if [ $major -lt 4 -o \( $major -eq 4 -a $minor -lt 2 \) ]; then
     exit 2
 elif [ $SKIP_VERSION_CHECK -eq 1 ]; then
     echo "Skipping OCaml version check"
-elif [ $major -ne 4 -o $minor -ne 10 ]; then
+elif [ $major -ne 4 -o $minor -ne 11 ]; then
     echo "This version of Camlp4 is for OCaml 4.10 but you are using OCaml $ocaml_version."
```
Comment (gasche) [inline configure]:
```suggestion
    echo "This version of Camlp4 is for OCaml 4.11 but you are using OCaml $ocaml_version."
```

**Comment by ygrek** [comment]
Thanks!

**Comment by kit-ty-kate** [comment]
@ygrek the first release candidate is out and the release is set for next week. Would it be ok to make a release soon so that people can have access to it before the release?

**Comment by ygrek** [comment]
thanks for the heads up but I was too slow :]
https://github.com/ocaml/opam-repository/pull/17026

**Pull request merged**